src/ethereum_test_forks/forks/forks.py

```diff
@@ -948,6 +944,29 @@ def max_blobs_per_block(cls, block_number: int = 0, timestamp: int = 0) -> int:
         """Blobs are enabled starting from Cancun, with a static max of 6 blobs."""
         return 6
 
+    @classmethod
+    def blob_schedule(cls, block_number: int = 0, timestamp: int = 0) -> BlobSchedule | None:
+        """
+        At Cancun, the fork object runs this routine to get the updated blob
+        schedule.
+        """
+        parent_fork = cls.parent()
+        assert parent_fork is not None, "Parent fork must be defined"
+        blob_schedule = parent_fork.blob_schedule(block_number, timestamp)
+        if blob_schedule is None:
+            last_blob_schedule = None
+            blob_schedule = BlobSchedule()
+        else:
+            last_blob_schedule = blob_schedule.last()
+        current_blob_schedule = ForkBlobSchedule(
+            target_blobs_per_block=cls.target_blobs_per_block(block_number, timestamp),
+            max_blobs_per_block=cls.max_blobs_per_block(block_number, timestamp),
+            base_fee_update_fraction=cls.blob_base_fee_update_fraction(block_number, timestamp),
+        )
+        if last_blob_schedule is None or last_blob_schedule != current_blob_schedule:
+            blob_schedule.append(fork=cls.__name__.lower(), schedule=current_blob_schedule)
```

Comment:
The EIP uses lowercase for this object, but the inconsistency between the network field (capitalized) and blob schedule key (lowercase) really stands out now, for example:
https://github.com/ethereum/execution-spec-tests/blob/d4b5894ad767b03d89750583cac74d3cac03a0b7/src/ethereum_test_specs/tests/fixtures/chainid_cancun_blockchain_test.json#L131-L136

Can we take the capitalized fork name for consistency?
```suggestion
            blob_schedule.append(fork=cls.__name__, schedule=current_blob_schedule)
```

Reply:
Agreed, let's change it for consistency.